reference/constraints/Url.rst

```diff
@@ -299,5 +300,78 @@ option to ``true``:
             }
         }
 
+dnsMessage
+~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``dnsMessage`` option was introduced in Symfony 2.7.
+
+**type**: ``string`` **default**: ``The host could not be resolved.``
+
+This message is shown when the ``checkDNS`` option is set to ``true`` and the
+DNS check failed.
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/AppBundle/Entity/Author.php
+        namespace AppBundle\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Author
+        {
+            /**
+             * @Assert\Url(
+             *    dnsMessage = "The host '{{ value }}' could not be resolved"
+             * )
+             */
+             protected $bioUrl;
+        }
+
+    .. code-block:: yaml
+
+        # src/AppBundle/Resources/config/validation.yml
+        AppBundle\Entity\Author:
+            properties:
+                bioUrl:
+                    - Url: { dnsMessage: The host "{{ value }}" could not be resolved }
```

Comment:
Ah, and what do you think to end the message with a period (in all examples) to have a complete sentence?